src/main.js

```diff
@@ -323,6 +332,18 @@ function getConfig(request) {
   .setHelpText("Do you want to reset Auth?")
   .setIsDynamic(true);
   
+  config.newInfo()
+  .setId('Caching Credentials')
+  .setText('You have set credentials to be cached: ' +  user.getProperty('dscc.areCredentialsCached'));
+  
+  config
+  .newSelectSingle()
+  .setId('cache_credentials')
+  .setName('Would you like to save your credentials?')
+  .setHelpText('Chose whether to cache your ODK username and password to avoid getting logged out every 24 hours. You can leave this blank if it is set to what you want it to be.')
```

Comment:
How about adding a note to clarify that this means their Central password is stored on Google servers? Maybe something like "This means your Central password will be stored on Google servers. We recommend using a Project Viewer account."

> You can leave this blank if it is set to what you want it to be.
Not immediately understanding this! 'no' will be selected by default, right? So it won't be blank?

Reply:
Oooh, I can definitely add that! And I need to play around with the language, but google data studio doesn't allow default options for the dropdown. I have a textbox above the select that lists their current selection (caching or no caching) and if they don't select anything then it leaves their current selection. That is why I said they could leave it blank.

Reply:
I see! I had the wrong UI in my head -- I thought that was a dialog. Makes sense, then!